core/src/main/java/org/elasticsearch/monitor/os/OsStats.java

```diff
@@ -152,8 +156,9 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     public static class Cpu implements Streamable {
+
         short percent = -1;
-        double loadAverage = -1;
+        double[] loadAverage = new double[] { -1, -1, -1 };
```

Comment:
@rmuir Removed in b50cdd11e27ca6da56bd2a1827e76a0bdb9fdad3

> I dont understand why we have readFrom() at all, vs. say using an abstract class with a ctor that takes Streamable, and thats just how things are deserialized. Then these various classes could be immutable and there would be much less confusion?

I think that we want to go in that direction with all of these eventually.